content/about/notes/1.1.1/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Istio 1.1.1
+publishdate: 2019-03-22
+icon: notes
+---
+
+Istio 1.1.1 is the first in a weekly series of patch releases for Istio 1.1.  It contains a small number of bug fixes and minor enhancements.
+
+{{< relnote_links >}}
+
+## Bug Fixes and Minor Enhancements
+
+- Configure Prometheus to monitor citadel ([Issue 12175](https://github.com/istio/istio/pull/12175))
+- Improve output of `istioctl's` verify install command ([Issue 12174](https://github.com/istio/istio/pull/12174))
```

Comment:
Improve the output of the `istioctl verify install` command (and make that a link)